moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js

```diff
@@ -97,13 +97,21 @@ const ProjectSummaryProjectDescription = ({
               fullWidth
               multiline={true}
               minRows={4}
+              error={description.length < 1}
               id="moped-project-description"
               label={null}
               onChange={handleProjectDescriptionChange}
               value={description}
+              helperText={
+                description.length < 1 ? "Description cannot be blank" : ""
```

Comment:
🙌

Reply:
should users be able to type in just spaces/newlines or should we handle that as well?

Reply:
I pushed up a change @roseeichelmann , do you think this works?